raft/raft_test.go

```diff
@@ -1300,10 +1300,14 @@ func TestUnreachable(t *testing.T) {
 
 	// set node 2 to unreachable
 	r.prs[2].Match = 3
-	r.prs[2].Next = 4
+	r.prs[2].Next = 5
 	r.prs[2].Wait = 0
 	r.prs[2].unreachable()
 
+	if wnext := r.prs[2].Match + 1; r.prs[2].Next != wnext {
```

Comment:
we only call unreachable at line 1305, so i test whether it sets the next field correctly here.